src/applications/hca/tests/00-all-fields.e2e.spec.js

```diff
@@ -5,9 +5,13 @@ const testData = require('./schema/maximal-test.json');
 const ENVIRONMENTS = require('site/constants/environments');
 const FormsTestHelpers = require('platform/testing/e2e/form-helpers');
 
+const vetDoBArr = testData.data.veteranDateOfBirth.split('-');
```

Comment:
just a thought: you could use array destructing here to store the year, month, and day in their own vars which would make it a little nicer to read when you use the values in the `expect` calls.

Reply:
Thx!  Done.